src/ILAMB/Confrontation.py

```diff
@@ -178,14 +188,14 @@ def __init__(self,**keywords):
                     self.lbls = ["site%d" % s for s in range(len(dataset.dimensions["data"]))]
             if "time" in dataset.dimensions:
                 t = dataset.variables["time"]
-                tdata = t[[0,-1]]
                 if "bounds" in t.ncattrs():
-                    tdata = dataset.variables[t.bounds]
-                    tdata = [tdata[0,0],tdata[-1,1]]
-                tdata = cf.num2date(tdata,units=t.units,calendar=t.calendar)
-                y0 = tdata[0].year
-                yf = tdata[1].year
-                
+                    t  = dataset.variables[t.bounds][...]
```

Comment:
Was what I had in here not working for some data? The issue is that your math here assumes that the calendar is `noleap` and the datum is `1850-01-01` and that isn't always the case at all. In the future version of ILAMB this will matter less as we will use xarray as the guts.

Reply:
This is also a carryover from ILAMB 2.5.